autotest.pm

```diff
@@ -148,6 +204,27 @@ sub loadtest ($script, %args) {
             EOM
         $is_python = 1;
     }
+    elsif ($script =~ m/\.lua$/) {
+        _debug_lua_version(); # this loads (actually defines) lua_eval()
+        $code .= <<~"EOM";
```

Comment:
Maybe
```suggestion
        $code .= sprintf <<~'EOM', $script_path;
```
then we don't need those backslashes

Reply:
What backslashes and what does the sprintf here do and why?

Reply:
I made a complete suggestion now below